src/main/c/Include/jep_platform.h

```diff
@@ -31,6 +31,18 @@
  * This file aims to be for Jep what pyport.h is for CPython.
  */
 
+/**
+ * If enabled Compile against the limited API, for version 3.8
+ *
+ * NOTE: This is higher than the minimum python version we normally require.
+ * However, the Limited API has been slowly expanded over time and we need to
+ * make use of the latest functionality. Almost all the "new" features in the Stable API
+ * are still present on old versions, only unstable.
+ *
+ * TODO: Make this configuration optional
+ */
+#define Py_LIMITED_API 0x03080000
```

Comment:
This is a hack.

Arguably, we could make this some sort of build flag given in the makfile or `setup.py`.
However, it's too unstable and feature-incomplete to ever become the default, so why bother?